### PR TITLE
Markers

### LINKED ISSUE #387: [Feature] List, search, filter markers

### LINKED ISSUE #1285: Add marker actors
By default, the scene's actors should be applied to the marker, unless set otherwise

### PR DESCRIPTION
**Pull request by boi123212321**
- close #1285 
- close #387

### CODE REVIEW

**Comment by boi123212321** [comment]
need https://github.com/porn-vault/porn-vault/pull/1384/files

File: app/src/views/Markers.vue [hunk 1]
```diff
@@ -256,11 +295,8 @@ export default class MarkerList extends mixins(DrawerMixin) {
   refreshed = true;
 
   resetPagination() {
-    markerModule.resetPagination();
-    this.refreshed = true;
-    this.loadPage(this.page).catch(() => {
-      this.refreshed = false;
-    });
+    this.searchState.page = 1;
+    this.updateRoute(this.searchStateManager.toQuery());
```
Comment (leadwolf) [inline app/src/views/Markers.vue:299]:
```suggestion
    this.searchStateManager.onValueChanged("page", 1);
    this.updateRoute(this.searchStateManager.toQuery(), false, () => {
      // If the query wasn't different, just reset the flag
      this.searchStateManager.refreshed = true;
    });
```

File: app/src/views/Markers.vue [hunk 2]
```diff
@@ -350,13 +400,13 @@ export default class MarkerList extends mixins(DrawerMixin) {
   }
 
   refreshPage() {
-    this.loadPage(markerModule.page);
+    this.loadPage();
   }
```
Comment (leadwolf) [inline app/src/views/Markers.vue:404]: can be deleted

File: app/src/views/Markers.vue [hunk 3]
```diff
@@ -374,13 +424,30 @@ export default class MarkerList extends mixins(DrawerMixin) {
...
 
-  mounted() {
-    if (!this.markers.length) {
-      this.refreshPage();
+  updateRoute(query: { [x: string]: string }, replace = false, noChangeCb: Function | null = null) {
+    if (isQueryDifferent(query, this.$route.query as Dictionary<string>)) {
+      // Only change the current url if the new url will be different to avoid redundant navigation
+      const update = {
+        name: "scenes",
```
Comment (leadwolf) [inline app/src/views/Markers.vue:431]:
```suggestion
        name: "markers",
```

**Review by leadwolf — APPROVED** [review]

**Pull request merged**